audnauseum/gui/ui.py

```diff
@@ -204,3 +209,67 @@ def show_popup(ui):
     msg.setText("A loop must be loaded.")
 
     msg.exec_()
+
+
+def transport_status(ui, looper: Looper, status):
+
+    if not looper.state == LooperStates.IDLE:
+
+        if status == 'record':
```

Comment:
I wonder if it would be better to check the looper state here instead? Though there are two Looper states that represent recording.